spec/models/form/sales/questions/outright_ownership_type_spec.rb

```diff
@@ -43,4 +41,31 @@
       "othtype" => [12],
     })
   end
+
+  it "has the correct header" do
+    expect(question.header).to eq("What is the type of outright sale?")
+  end
+
+  describe "partial guidance" do
+    context "when the form is for year 2023" do
+      let(:start_date) { Time.utc(2023, 2, 8) }
+
+      it "has the correct guidance_partial" do
+        expect(question.guidance_partial).to eq("outright_sale_type_definitions")
+      end
+
+      it "is at the top" do
+        expect(question.top_guidance?).to eq(true)
+        expect(question.bottom_guidance?).to eq(false)
+      end
+    end
+
+    context "when the form is for before year 2023" do
+      let(:start_date) { Time.utc(2022, 2, 8) }
+
+      it "does not display a guidance partial" do
+        expect(question.guidance_partial).to eq(nil)
+      end
+    end
```

Comment:
my personal preference for these things, for complete clarity and reduction of any cognitive load parsing dates for the next reader, is to to name these contexts something like 23/24 rather than just the start year.

lines 50/51 I think show this as when there are other places around the codebase where we use one year only, this tends to refer to the start year, whereas (2023, 2, 8) is in the 22/23 year.

this therefore warrants investigation as I would expect this test to fail if the guidance is only meant to be displayed from 23/24 onwards.